lifecycle-application/pom.xml

```diff
@@ -40,8 +40,7 @@
             </activation>
             <properties>
                 <!-- please keep Mandrel and RHBQ version compatible -->
-                <quarkus.platform.version>2.7.6.Final</quarkus.platform.version>
-                <quarkus.native.builder-image>quay.io/quarkus/ubi-quarkus-mandrel:22.1-java11</quarkus.native.builder-image>
```

Comment:
This property is not used. 
Was introduced in a previous commit:  https://github.com/quarkus-qe/quarkus-test-suite/pull/786/files